test/bdd/bddtests_test.go

```diff
@@ -29,6 +29,11 @@ func TestMain(m *testing.M) {
 	tags := "all"
 	flag.Parse()
 
+	format := "progress"
+	if getCmdArg("test.v") == "true" {
```

Comment:
It would be nice if we could pass a flag to make as well.

(e.g., make bdd-test -v)

Reply:
it set by default